### PR TITLE
Update Jupyter docs to reflect the move to ksonnet

### PR DESCRIPTION
**Pull request by jlewi**
* Update the docs to provide instructions using ksonnet
* Remove the old YAML manifests since we now use ksonnet

### CODE REVIEW

**Comment by foxish** [comment]
Do we want to remove the old YAML config and docs entirely? Is it because of the additional effort that might be needed to keep the ksonnet config and YAML files in sync?

**Comment by foxish** [comment]
cc @kow3ns

**Comment by jlewi** [comment]
I think we want to remove them because it creates confusion as well as a maintenance headache.

If people just want to see the manifests they can use `ks show` to generate them.

**Comment by foxish** [comment]
Ok. Maybe we can add the `ks show` command to the docs then.

File: components/jupyterhub/README.md [hunk 1]
```diff
@@ -42,7 +32,14 @@ The above will expose JupyterHub on http://localhost:8000. The pod name can be o
...
 
-Configuration for JupyterHub is shipped separately and contained within the configmap under `manifests/config.yaml`. It is a Python file that is consumed by JupyterHub on starting up. The supplied configuration has reasonable defaults for the requisite fields and **no authenticator** configured by default.
+Configuration for JupyterHub is shipped separately and contained within the configmap defined by the [core componenent](https://github.com/google/kubeflow/tree/master/kubeflow). It is a Python file that is consumed by JupyterHub on starting up. The supplied configuration has reasonable defaults for the requisite fields and **no authenticator** configured by default. Furthermore, we provide a number of parameters that can be used to configure
+the core component. To see a list of ksonnet parameters run
+
+```
+ks prototype describe kubeflow-core
+```
+
+If the provided parameters don't provide the flexibility you need, you can take advantage of ksonnet to customize the core component and use a config file fully specified by you.
```
Comment (foxish) [inline components/jupyterhub/README.md]: Maybe specify how to get the manifest here?

Comment (jlewi) [reply components/jupyterhub/README.md]: Filed #56

**Comment by jlewi** [comment]
I added instructions on how to use ks show to the user guide so people can inspect the manifest.

Since this is a trivial changes I'm going to use my super powers to merge it.

**Pull request merged**